pyproject.toml

```diff
@@ -121,6 +121,7 @@ skip_glob = ["typings/*", "docs/*"]
 pythonVersion = "3.10"
 include = ["bentoml"]
 exclude = ['bentoml/_version.py','bentoml/__main__.py']
+useLibraryCodeForTypes = true
```

Comment:
note from: https://github.com/microsoft/pyright/blob/main/docs/settings.md

```
pyright.useLibraryCodeForTypes [boolean]: This setting is deprecated in favor of python.analysis.useLibraryCodeForTypes. It will be removed at a future time.
```